bin/all_sky_search/pycbc_sngls_statmap

```diff
@@ -153,6 +146,42 @@ bg_far, fg_far = significance.get_far(
     fg_time,
     **significance_dict[ifo])
 
+fg_far = significance.apply_far_limit(fg_far, significance_dict, combo=ifo)
+bg_far = significance.apply_far_limit(bg_far, significance_dict, combo=ifo)
+
+bg_ifar = 1. / bg_far
+fg_ifar = 1. / fg_far
+
+f['background/ifar'] = conv.sec_to_year(bg_ifar)
+
+f.attrs['background_time'] = fg_time
+f.attrs['foreground_time'] = fg_time
+
+# Find foreground triggers with IFAR > the set limit and remove from
+# the exclusive background
+
+# Need to make copies for use as exclusive triggers as these will have
+# items removed from them, and don't want to overwrite the original
+fg_time_exc = copy.deepcopy(fg_time)
```

Comment:
is this actually a float rather than a list/array?

Reply:
yes it is, so doesnt need the copy
```suggestion
fg_time_exc = fg_time
```